src/goto-symex/goto_symex.h

```diff
@@ -385,22 +517,62 @@ class goto_symext
   static exprt add_to_lhs(const exprt &lhs, const exprt &what);
 
   virtual void symex_gcc_builtin_va_arg_next(
-    statet &, const exprt &lhs, const side_effect_exprt &);
+    statet &state, const exprt &lhs, const side_effect_exprt &code);
+  /// Symbolically execute an assignment instruction that has an `allocate` on
+  /// the right hand side
+  /// \param state SymEx state for current instruction
+  /// \param lhs The expression to assign to
+  /// \param code The `allocate` expression
   virtual void symex_allocate(
-    statet &, const exprt &lhs, const side_effect_exprt &);
-  virtual void symex_cpp_delete(statet &, const codet &);
+    statet &state, const exprt &lhs, const side_effect_exprt &code);
+  /// Symbolically execute an OTHER instruction that does a CPP `delete`
+  /// \param state SymEx state for current instruction
+  /// \param code The cleaned up CPP `delete` instruction
+  virtual void symex_cpp_delete(statet &state, const codet &code);
+  /// Symbolically execute an assignment instruction that has a CPP `new` or
+  /// `new array` or a Java `new array` on the right hand side
+  /// \param state SymEx state for current instruction
+  /// \param lhs The expression to assign to
+  /// \param code The `new` expression
   virtual void symex_cpp_new(
-    statet &, const exprt &lhs, const side_effect_exprt &);
-  virtual void symex_fkt(statet &, const code_function_callt &);
-  virtual void symex_trace(statet &, const code_function_callt &);
-  virtual void symex_printf(statet &, const exprt &rhs);
-  virtual void symex_input(statet &, const codet &);
-  virtual void symex_output(statet &, const codet &);
-
+    statet &state, const exprt &lhs, const side_effect_exprt &code);
+  /// Symbolically execute a FUNCTION_CALL instruction for a function whose
+  /// name starts with CPROVER_FKT_PREFIX
+  /// \remarks
+  /// While the name seems to imply that this would be called when SymEx
+  /// doesn't know what to do, it may actually be derived from a German
+  /// abbreviation for function.
```

Comment:
Looool. It had never occurred to me that this was not an appropriate English abbreviation at all... (But also I think it's just something in TG blocking us from removing it completely.)